### PR TITLE
Small logging fixes

### PR DESCRIPTION
**Pull request by bbrks**
- Added missing escapes for Windows path backslashes

### CODE REVIEW

File: modules/ROOT/pages/logging.adoc [hunk 1]
```diff
@@ -202,7 +202,7 @@ On MS Windows `logFilePath` supports the following path formats.
 [source,javascript]
 ----
 "C:/var/tmp/sglogfile.log"
-`C:\var\tmp\sglogfile.log`
+`C:\\var\\tmp\\sglogfile.log`
 `/var/tmp/sglogfile.log`
 "/var/tmp/sglogfile.log"
```
Comment (bbrks) [inline modules/ROOT/pages/logging.adoc:207]: @jamiltz I'm not sure if this is a valid Windows log file path or not. I can see how omitting the leading `/` would make it relative to SG, but can't say without testing.

Comment (jamesnocentini) [reply modules/ROOT/pages/logging.adoc:207]: It looks like Windows absolute paths can start with a leading `\` https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats

Does that clarify? (from what I understand `\\` == `/` on windows?)

File: modules/ROOT/pages/logging.adoc [hunk 2]
```diff
@@ -202,7 +202,7 @@ On MS Windows `logFilePath` supports the following path formats.
 [source,javascript]
 ----
 "C:/var/tmp/sglogfile.log"
-`C:\var\tmp\sglogfile.log`
+`C:\\var\\tmp\\sglogfile.log`
```
Comment (jamesnocentini) [inline modules/ROOT/pages/logging.adoc:205]: A single backslash seems valid looking at https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats

Am I missing something on this page?

File: modules/ROOT/pages/logging.adoc (hunk 1 above)
Comment (bbrks) [reply modules/ROOT/pages/logging.adoc:207]: Oops, I'd not noticed the backticks around the 2nd line - You only need to escape backslashes in double quotes in JSON. This PR can be closed! :) 

The leading slash makes sense to me after reading that doc - Go will translate forward slashes to Windows-specific backslashes - so should work fine!

**Comment by bbrks** [comment]
Closing - see comment

File: modules/ROOT/pages/logging.adoc (hunk 1 above)
Comment (jamesnocentini) [reply modules/ROOT/pages/logging.adoc:207]: 👍

**Pull request closed without merging**